src/screens/Address/GroupSelectScreen.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { TOTAL_NUMBER_OF_GROUPS } from '@alephium/web3'
+import { StackScreenProps } from '@react-navigation/stack'
+import React from 'react'
+
+import { SelectOption } from '~/components/inputs/Select'
+import BoxSurface from '~/components/layout/BoxSurface'
+import { ScreenSection } from '~/components/layout/Screen'
+import ScrollScreen from '~/components/layout/ScrollScreen'
+import RadioButtonRow from '~/components/RadioButtonRow'
+import { useNewAddressContext } from '~/contexts/NewAddressContext'
+import { NewAddressNavigationParamList } from '~/navigation/NewAddressNavigation'
+
+type ScreenProps = StackScreenProps<NewAddressNavigationParamList, 'GroupSelectScreen'>
+
+const groupSelectOptions: SelectOption<number | undefined>[] = Array.from(Array(TOTAL_NUMBER_OF_GROUPS)).map(
+  (_, index) => ({
+    value: index,
+    label: `Group ${index}`
+  })
+)
+
+groupSelectOptions.unshift({
+  value: undefined,
+  label: 'Default'
+} as SelectOption<undefined>)
```

Comment:
Unnecessary if we use the method proposed before.

Reply:
Removed ;)